objective-c/HelloWorld/HelloWorld/ViewController.m

```diff
@@ -0,0 +1,27 @@
+//
+//  ViewController.m
+//  HelloWorld
+//
+//  Created by Michael Lumish on 6/11/15.
+//  Copyright (c) 2015 Google. All rights reserved.
+//
+
+#import "ViewController.h"
+
+@interface ViewController ()
+
+@end
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+  // Do any additional setup after loading the view, typically from a nib.
+}
+
+- (void)didReceiveMemoryWarning {
+  [super didReceiveMemoryWarning];
+  // Dispose of any resources that can be recreated.
+}
```

Comment:
You can delete it's .h if you move the interface declaration here, as nobody else is importing it. But as there's no code to show here anyway, the same logic as before applies: you can just put the files in a subdir.